Sources/StreamChat/Config/ChatClientConfig.swift

```diff
@@ -190,6 +190,9 @@ public struct ChatClientConfig {
     /// The `URLSessionConfiguration` being used as default configuration for the `APIClient` and
     /// `WebSocketClient`
     public var urlSessionConfiguration: URLSessionConfiguration = .default
+    
+    /// How many hours the unsent messages should be queued for sending when the internet connection is available.
+    public var unsentMessagesRetryHoursThreshold: CGFloat = 12
```

Comment:
What about using `Max` instead of `Retry`? I think "Max" it is more clear 🤔 Retry it feels like it will retry every 12 hours 🤔 
```suggestion
    public var unsentMessagesMaxHoursThreshold: CGFloat = 12
```

Reply:
`unsentMessagesMaxHoursThreshold` doesn't really mean anything TBH. What is the it's the max threshold (expressed in hours) of what exactly?

Reply:
It was renamed into `queuedActionsMaxHoursThreshold`. Open to other suggestions as well.